lib/activerecord-bitemporal.rb

```diff
@@ -107,10 +107,16 @@ def transaction_from_cannot_be_greater_equal_than_transaction_to
   def bitemporalize(
     enable_strict_by_validates_bitemporal_id: false,
     enable_default_scope: true,
-    enable_merge_with_except_bitemporal_default_scope: false
+    enable_merge_with_except_bitemporal_default_scope: false,
+    valid_from_key: :valid_from,
+    valid_to_key: :valid_to
   )
+    return if @bitemporalized
+    @bitemporalized = true
```

Comment:
might want to use `ancestors.include?` instead of `@bitemporalized`.
```suggestion
    return if ancestors.include? ActiveRecord::Bitemporal
```

Reply:
I'm wondering if using `ancestors.include?` could cause the subsequent logic to be skipped when the module is included as usual—what do you think?

```rb
module Bitemporal
  extend ActiveSupport::Concern

  included do
    bitemporalize
  end
end

module Bitemporal::Bitemporalize
  def bitemporalize
    if ancestors.include? Bitemporal
      puts "Bytemporal is included"
      return
    end

    puts "Bytemporal is not included"
    puts "do other processes"
    include Bitemporal
  end
end

class BaseClass
end

BaseClass.extend(Bitemporal::Bitemporalize)

class ModelClass < BaseClass
  include Bitemporal
  # => expected: "Bytemporal is not included" and "do other processes"
  # => actual: "Bytemporal is included"
end
```

Reply:
I see, Let's use `InstanceMethods` .
```suggestion
    return if ancestors.include? InstanceMethods
```
In any case, I don't think we should use instance variables.